core-bundle/src/EventListener/BackendMenuListener.php

```diff
@@ -93,6 +139,139 @@ public function onBuild(MenuEvent $event): void
+            ->setExtra('safe_label', true)
+            ->setExtra('translation_domain', false)
+        ;
+
+        $tree->addChild($alerts);
+
+        $preview = $factory
+            ->createItem('preview')
+            ->setLabel('MSC.fePreview')
+            ->setUri($this->previewUrlGenerator->getPreviewUrl())
+            ->setLinkAttribute('class', 'icon-preview')
+            ->setLinkAttribute('title', $this->trans('MSC.fePreviewTitle'))
+            ->setLinkAttribute('target', '_blank')
+            ->setLinkAttribute('accesskey', 'f')
+            ->setExtra('translation_domain', 'contao_default')
+        ;
+
+        $tree->addChild($preview);
+
+        $submenu = $factory
+            ->createItem('submenu')
+            ->setLabel($this->trans('MSC.user').' '.$user->username)
+            ->setAttribute('class', 'submenu')
+            ->setLabelAttribute('class', 'h2')
+            ->setExtra('translation_domain', false)
+        ;
+
+        $tree->addChild($submenu);
+
+        $info = $factory
+            ->createItem('info')
+            ->setLabel(sprintf('<strong>%s</strong> %s', $user->name, $user->email))
+            ->setAttribute('class', 'info')
+            ->setExtra('safe_label', true)
+            ->setExtra('translation_domain', false)
+        ;
+
+        $submenu->addChild($info);
+
+        $login = $factory
+            ->createItem('login')
+            ->setLabel('MSC.profile')
+            ->setUri($this->router->generate('contao_backend', ['do' => 'login', 'ref' => $ref]))
+            ->setLinkAttribute('class', 'icon-profile')
+            ->setExtra('translation_domain', 'contao_default')
+        ;
+
+        $submenu->addChild($login);
+
+        $security = $factory
+            ->createItem('security')
+            ->setLabel('MSC.security')
+            ->setUri($this->router->generate('contao_backend', ['do' => 'security', 'ref' => $ref]))
+            ->setLinkAttribute('class', 'icon-security')
+            ->setExtra('translation_domain', 'contao_default')
+        ;
+
+        $submenu->addChild($security);
+
+        $logout = $factory
+            ->createItem('logout')
+            ->setLabel($this->getLogoutLabel())
+            ->setUri($this->logoutUrlGenerator->getLogoutUrl())
+            ->setLinkAttribute('class', 'icon-logout')
+            ->setLinkAttribute('accesskey', 'q')
+            ->setExtra('translation_domain', false)
+        ;
+
+        $submenu->addChild($logout);
+
+        $buger = $factory
+            ->createItem('burger')
+            ->setLabel('<button type="button" id="burger"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="#fff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M3 12h18M3 6h18M3 18h18"/></svg></button>')
```

Comment:
The button should have a label or title, so we should probably not remove `MSC.burgerTitle`

Reply:
I don't think so, because the button is only visible on mobile devices and neither title nor label would be shown.

Reply:
I would assume it is visible to screen readers, and most likely to an iPhone screen reader as well